.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -14,7 +14,7 @@ Check these items off in the GitHub PR UI as you complete them.
 
 ## Relevant links
 
-- [Preview link](url) 🔎
+- [Preview link](https://dev-portal-git-BRANCH_NAME-hashicorp.vercel.app/PATH_TO_VIEW) 🔎
```

Comment:
@kendallstrautman Could you add a note via MD comment about how Vercel removes `.` characters from branch names? This was something I had no idea about before I joined, I think it'd be valuable to mention.

```suggestion
<!--
Comment about the `.` character in branch names
-->

- [Preview link](https://dev-portal-git-BRANCH_NAME-hashicorp.vercel.app/PATH_TO_VIEW) 🔎
```

Reply:
Good call!